src/main/java/frc/robot/Constants.java

```diff
@@ -5,6 +5,49 @@
 package frc.robot;
 
 public final class Constants {
+  
+  //Shooter
+  public static final int SHOOTER_MOTOR_ID_1 = 44;
+  public static final int SHOOTER_MOTOR_ID_2 = 45;
+
+  public static final int SHOOTER_SENSOR_ID_1 = 0;
+  public static final int SHOOTER_SENSOR_ID_2 = 1;
+
+  public static final double SHOOTER_MOTOR_SPEED = 0.75;
+
+  public static final double SHOOTER_MOTOR_1_RPM = 12000;
+  public static final double SHOOTER_MOTOR_2_RPM = 12000;
+
+  //Shooter motor 1 PID constants
+  public static final double SHOOTER_MOTOR_1_PID_P = 5e-5;
```

Comment:
The PID values for both motors will be the same because it is the same type of motor and motor controller, so you can have a SHOOTER_MOTOR_PID_P (and all the other values) and use it for both motors. The only value that may be different between the 2 motors is the RPM.